docs: improve docs about how to work with logs (#3626)
docker-mailserver/docker-mailserver: docs/content/config/debugging.md

4. **Understand the basics of mail servers**: Especially for beginners, make sure you read our [Introduction][docs-introduction] and [Usage][docs-usage] articles.
5. **Search the whole FAQ**: Our [FAQ][docs-faq] contains answers for common problems. Make sure you go through the list.
6. **Reduce the scope**: Ensure that you can run a basic setup of DMS first. Then incrementally restore parts of your original configuration until the problem is reproduced again. If you're new to DMS, it is common to find the cause is misunderstanding how to configure a minimal setup.

### Debug a running container

#### General

To get a shell inside the container run: `docker exec -it <CONTAINER NAME> bash`. To install additional software, run:

1. `apt-get update` to update repository metadata.
2. `apt-get install <PACKAGE>` to install a package, e.g., `apt-get install neovim` if you want to use NeoVim instead of `nano` (which is shipped by default).

#### Logs

If you need more flexibility than what the `docker logs` command offers, then the most useful locations to get relevant DMS logs within the container are:

> - `/var/log/mail/mail.log`,

Reviewer: ```suggestion
- `/var/log/mail/mail.log`
```

> If you need more flexibility than `docker logs` offers, within the container
> 
> 1. `/var/log/mail/mail.log`,
> 2. `/var/log/mail/mail/<SERVICE>.log`, and
> 3. `/var/log/supervisor/<SERVICE>.log`

Reviewer: ```suggestion
If you need more flexibility than what the `docker logs` command offers, then the most useful locations to get relevant DMS logs within the container are:

- `/var/log/mail/mail.log`,
- `/var/log/mail/mail/<SERVICE>.log`
- `/var/log/supervisor/<SERVICE>.log`
```

> are the most useful locations to get relevant DMS logs. You may use `nano` (a text editor), `less` (a file viewer) or `tail`/`cat` to inspect (or edit) their contents.

Reviewer: ```suggestion
You may use `nano` (a text editor) to edit files, while `less` (a file viewer) and `tail`/`cat` are useful tools to inspect the contents of logs.
```
